### PR TITLE
fix typos

### PR DESCRIPTION
**Pull request by Takashiidobe**

### CODE REVIEW

File: content/en/docs/reference/config-api/apiserver-audit.v1.md [hunk 1]
```diff
@@ -253,7 +253,7 @@ be specified per rule in which case the union of both are omitted.</p>
 <td>
    <p>OmitManagedFields indicates whether to omit the managed fields of the request
 and response bodies from being written to the API audit log.
-This is used as a global default - a value of 'true' will omit the managed fileds,
+This is used as a global default - a value of 'true' will omit the managed fields,
```
Comment (windsonsea) [inline content/en/docs/reference/config-api/apiserver-audit.v1.md:256]: Check the top text in this `reference` file. It is auto-generated and do not edit.

File: content/en/docs/reference/config-api/kubeadm-config.v1beta4.md [hunk 2]
```diff
@@ -16,7 +16,7 @@ Use <code>APIServer.ExtraEnvs</code>, <code>ControllerManager.ExtraEnvs</code>,
 <code>Etcd.Local.ExtraEnvs</code>.</li>
 <li>The <code>ResetConfiguration</code> API type is now supported in v1beta4.
 Users are able to reset a node by passing a <code>--config</code> file to <code>kubeadm reset</code>.</li>
-<li><code>dry-run</code> mode is now configureable in InitConfiguration and JoinConfiguration config files.</li>
+<li><code>dry-run</code> mode is now configurable in InitConfiguration and JoinConfiguration config files.</li>
```
Comment (windsonsea) [inline content/en/docs/reference/config-api/kubeadm-config.v1beta4.md:19]: ditto

File: content/en/docs/reference/instrumentation/metrics.md [hunk 3]
```diff
@@ -1482,14 +1482,14 @@ Alpha metrics do not have any API guarantees. These metrics must be used at your
 	</ul>
 	</div><div class="metric" data-stability="alpha">
 	<div class="metric_name">ephemeral_volume_controller_create_failures_total</div>
-	<div class="metric_help">Number of PersistenVolumeClaims creation requests</div>
+	<div class="metric_help">Number of PersistentVolumeClaims creation requests</div>
```
Comment (windsonsea) [inline content/en/docs/reference/instrumentation/metrics.md:1485]: ditto, auto-generated. You can edit the upstream file.

File: content/en/docs/reference/kubernetes-api/cluster-resources/api-service-v1.md [hunk 4]
```diff
@@ -62,7 +62,7 @@ APIServiceSpec contains information for locating and communicating with a server
...
+  GroupPriorityMinimum is the priority this group should have at least. Higher priority means that the group is preferred by clients over lower priority ones. Note that other versions of this group might specify even higher GroupPriorityMinimum values such that the whole group gets a higher priority. The primary sort is based on GroupPriorityMinimum, ordered highest number to lowest (20 before 10). The secondary sort is based on the alphabetical comparison of the name of the object.  (v1.bar before v1.foo) We'd recommend something like: *.k8s.io (except extensions) at 18000 and PaaSes (OpenShift, Deis) are recommended to be in the 2000s
```
Comment (windsonsea) [inline content/en/docs/reference/kubernetes-api/cluster-resources/api-service-v1.md:65]: Do not edit this file.

File: content/en/docs/reference/kubernetes-api/config-and-storage-resources/csi-driver-v1.md [hunk 5]
```diff
@@ -59,68 +59,68 @@ CSIDriverSpec is the specification of a CSIDriver.
 - **attachRequired** (boolean)
 
   attachRequired indicates this CSI volume driver requires an attach operation (because it implements the CSI ControllerPublishVolume() method), and that the Kubernetes attach detach controller should call the attach volume interface which checks the volumeattachment status and waits until the volume is attached before proceeding to mounting. The CSI external-attacher coordinates with CSI volume driver and updates the volumeattachment status when the attach operation is complete. If the CSIDriverRegistry feature gate is enabled and the value is specified to false, the attach operation will be skipped. Otherwise the attach operation will be called.
-  
+
```
Comment (windsonsea) [inline content/en/docs/reference/kubernetes-api/config-and-storage-resources/csi-driver-v1.md:62]: ditto

File: content/en/docs/reference/kubernetes-api/config-and-storage-resources/csi-driver-v1.md [hunk 6]
```diff
@@ -15,7 +15,7 @@ The file is auto-generated from the Go source code of the component using a gene
 [generator](https://github.com/kubernetes-sigs/reference-docs/). To learn how
 to generate the reference documentation, please read
 [Contributing to the reference documentation](/docs/contribute/generate-ref-docs/).
-To update the reference content, please follow the 
+To update the reference content, please follow the
```
Comment (windsonsea) [inline content/en/docs/reference/kubernetes-api/config-and-storage-resources/csi-driver-v1.md:18]: ditto

File: content/en/docs/reference/kubernetes-api/service-resources/endpoint-slice-v1.md [hunk 7]
```diff
@@ -139,7 +139,7 @@ EndpointSlice represents a subset of the endpoints that implement a service. For
 
   - **ports.name** (string)
 
-    name represents the name of this port. All ports in an EndpointSlice must have a unique name. If the EndpointSlice is dervied from a Kubernetes service, this corresponds to the Service.ports[].name. Name must either be an empty string or pass DNS_LABEL validation: * must be no more than 63 characters long. * must consist of lower case alphanumeric characters or '-'. * must start and end with an alphanumeric character. Default is empty string.
+    name represents the name of this port. All ports in an EndpointSlice must have a unique name. If the EndpointSlice is derived from a Kubernetes service, this corresponds to the Service.ports[].name. Name must either be an empty string or pass DNS_LABEL validation: * must be no more than 63 characters long. * must consist of lower case alphanumeric characters or '-'. * must start and end with an alphanumeric character. Default is empty string.
```
Comment (windsonsea) [inline content/en/docs/reference/kubernetes-api/service-resources/endpoint-slice-v1.md:142]: ditto

**Review by windsonsea — COMMENTED** [review]
Do not directly edit the auto-generated files.

**Comment by Takashiidobe** [comment]
Thanks for the feedback, I've undone the changes to the files mentioned.

File: content/en/docs/reference/config-api/kubeconfig.v1.md
Comment (Gauravpadam) [inline content/en/docs/reference/config-api/kubeconfig.v1.md:1]: We should revert these changes, This is an auto-generated page

**Review by Gauravpadam — COMMENTED** [review]
Thanks for the PR

An lgtm from me; except the one change that is made to an auto-generated page

**Comment by Takashiidobe** [comment]
hopefully that's the last one, reverted the change to that file

**Comment by Takashiidobe** [comment]
@Gauravpadam if you have time for another look i'd much appreciate it

**Review by Gauravpadam — COMMENTED** [review]
Thanks

/lgtm

**Comment by reylejano** [comment]
This is a good PR and a good example on how to fix typos. @Takashiidobe went through multiple files to fix typos.
Thank you @Takashiidobe 
/approve
